examples/step-11/step-11.cc

```diff
@@ -188,7 +188,7 @@ namespace Step11
     // beforehand the maximal number of entries per row, either for all rows
     // or for each row separately. There are functions in the library which
     // can tell you this number in case you just have hanging node constraints
-    // (namely <code>DoFHandler::max_coupling_between_dofs</code>), but how is
+    // (namely <code>DoFHandler::max_couplings_between_dofs</code>), but how is
```

Comment:
```suggestion
    // (namely DoFHandler::max_couplings_between_dofs), but how is
```
This problem was already here, but since its going to be a hyperlink we don't need the HTML `<code>` and `</code>` delimiters.